protonvpn_cli/cli.py

```diff
@@ -52,19 +52,16 @@
 import shutil
```

Comment:
build.build.sh runs a fmt, yapf, and SA, mypy. All fmt changes where done by yapf. Extra comments were added by me as exceptions to mypy.